### PR TITLE
Create sprint-93-summary.md

### PR DESCRIPTION
**Pull request by reitermb**
## Summary of Changes
Adds sprint 93 summary

### CODE REVIEW

File: docs/Sprint-Review/sprint-93-summary.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+## Tickets
+### Completed/Merged
+* [#2781 As a developer,  I want to have documentation on django migration best practices](https://app.zenhub.com/workspaces/sprint-board-5f18ab06dfd91c000f7e682e/issues/gh/raft-tech/tanf-app/2781)
+* [#2813 Reduce dev environment count](https://app.zenhub.com/workspaces/sprint-board-5f18ab06dfd91c000f7e682e/issues/gh/raft-tech/tanf-app/2813)
+* [#2790 Kibana deployment](https://app.zenhub.com/workspaces/sprint-board-5f18ab06dfd91c000f7e682e/issues/gh/raft-tech/tanf-app/2790)
+* [#2824 Resolve issues raised in pen testing findings](https://app.zenhub.com/workspaces/sprint-board-5f18ab06dfd91c000f7e682e/issues/gh/raft-tech/tanf-app/2824)
+
+### Ready to Merge
+* [#1350 As an OFA staff member I want to access Kibana from TDP](https://github.com/raft-tech/TANF-app/issues/1350)
+* [#2757 Generate preparser errors when multirecord rows are the wrong length](https://github.com/raft-tech/TANF-app/issues/2757)
```
Comment (ADPennington) [inline docs/Sprint-Review/sprint-93-summary.md]: @reitermb this is still open.

File: docs/Sprint-Review/sprint-93-summary.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+
+
+### Submitted (QASP Review, OCIO Review)
+* [#2646 - Populate data file summary case aggregates differently per section](https://app.zenhub.com/workspaces/sprint-board-5f18ab06dfd91c000f7e682e/issues/gh/raft-tech/tanf-app/2646)
+* [#2820 [bug] Uncaught exception re: parsing error preventing feedback report generation](https://app.zenhub.com/workspaces/sprint-board-5f18ab06dfd91c000f7e682e/issues/gh/raft-tech/tanf-app/2820)
+* [#2746 As an STT, I need to know if there are issues with the DOBs reported in my data files](https://app.zenhub.com/workspaces/sprint-board-5f18ab06dfd91c000f7e682e/issues/gh/raft-tech/tanf-app/2746)
+* [#2729 As a developer, I want to move migration commands in the pipeline to CircleCI](https://app.zenhub.com/workspaces/sprint-board-5f18ab06dfd91c000f7e682e/issues/gh/raft-tech/tanf-app/2729)
+* [#2853 - Database Backup Bug](https://github.com/raft-tech/TANF-app/pull/2853)
+* [#2681 TANF Section 1 Validation cleanup](https://github.com/raft-tech/TANF-app/issues/2681)
```
Comment (ADPennington) [inline docs/Sprint-Review/sprint-93-summary.md:47]: @reitermb this is still in-progress.

File: docs/Sprint-Review/sprint-93-summary.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+        * Unblocks #1350 when complete 
+* DevOps:
+    * #2790 - Update deployment code to support Kibana and integrate with Standing Elastic instance
+* Design: 
+    * Tie up current documentation work
+    * Continue refinement of research roadmap
+
+
+## Tickets
+### Completed/Merged
```
Comment (ADPennington) [inline docs/Sprint-Review/sprint-93-summary.md:21]: @reitermb can you double-check [here](https://github.com/raft-tech/TANF-app/releases/tag/v3.3.0-Sprint-93) to confirm that everything is captured for this sprint?

**Review by andrew-jameson — APPROVED** [review]

**Review by lfrohlich — DISMISSED** [review]
Per @ADPennington 's comments, changes are requested

File: docs/Sprint-Review/sprint-93-summary.md (hunk 2 above)
Comment (reitermb) [reply docs/Sprint-Review/sprint-93-summary.md:47]: @ADPennington could you clarify which one? I'm seeing everything in Submitted here as also appearing to be in QASP in zenhub

**Comment by reitermb** [comment]
@ADPennington @lfrohlich 
I gave completed/merged a second pass given the [changelog](https://github.com/raft-tech/TANF-app/releases/tag/v3.3.0-Sprint-93). 

One item on there was a Spring 92 item and I added one final commit to that sprint summary accordingly. All else was added to this one or already accounted for under umbrella tickets like #2824

File: docs/Sprint-Review/sprint-93-summary.md [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+
+
+
+### Submitted (QASP Review, OCIO Review)
+* [#2646 - Populate data file summary case aggregates differently per section](https://app.zenhub.com/workspaces/sprint-board-5f18ab06dfd91c000f7e682e/issues/gh/raft-tech/tanf-app/2646)
+* [#2820 [bug] Uncaught exception re: parsing error preventing feedback report generation](https://app.zenhub.com/workspaces/sprint-board-5f18ab06dfd91c000f7e682e/issues/gh/raft-tech/tanf-app/2820)
+* [#2746 As an STT, I need to know if there are issues with the DOBs reported in my data files](https://app.zenhub.com/workspaces/sprint-board-5f18ab06dfd91c000f7e682e/issues/gh/raft-tech/tanf-app/2746)
+* [#2729 As a developer, I want to move migration commands in the pipeline to CircleCI](https://app.zenhub.com/workspaces/sprint-board-5f18ab06dfd91c000f7e682e/issues/gh/raft-tech/tanf-app/2729)
+* [#2853 - Database Backup Bug](https://github.com/raft-tech/TANF-app/pull/2853)
+* [#2681 TANF Section 1 Validation cleanup](https://github.com/raft-tech/TANF-app/issues/2681)
```
Comment (ADPennington) [inline docs/Sprint-Review/sprint-93-summary.md:47]: #2681 i believe.

File: docs/Sprint-Review/sprint-93-summary.md (hunk 2 above)
Comment (ADPennington) [reply docs/Sprint-Review/sprint-93-summary.md:47]: apologies i confused #2681 with #2838

File: docs/Sprint-Review/sprint-93-summary.md [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+* [#2813 Reduce dev environment count](https://app.zenhub.com/workspaces/sprint-board-5f18ab06dfd91c000f7e682e/issues/gh/raft-tech/tanf-app/2813)
+* [#2790 Kibana deployment](https://app.zenhub.com/workspaces/sprint-board-5f18ab06dfd91c000f7e682e/issues/gh/raft-tech/tanf-app/2790)
+* [#2824 Resolve issues raised in pen testing findings](https://app.zenhub.com/workspaces/sprint-board-5f18ab06dfd91c000f7e682e/issues/gh/raft-tech/tanf-app/2824)
+* [#2832 Accessibility Guide update](https://github.com/raft-tech/TANF-app/pull/2832)
+* [#2794 Make Case Number validators accept spaces and special characters](https://github.com/raft-tech/TANF-app/issues/2794)
+* [#2771 Update validateSSN to allow for all 0s, 1s,...9s as valid values except in cat3 scenarios](https://github.com/raft-tech/TANF-app/issues/2771)
+* [#2855 Update Failed Deployment Troubleshooting](https://github.com/raft-tech/TANF-app/pull/2855)
+* [#2768 Fix production OWASP scan reporting](https://app.zenhub.com/workspaces/sprint-board-5f18ab06dfd91c000f7e682e/issues/gh/raft-tech/tanf-app/2768)
```
Comment (ADPennington) [inline docs/Sprint-Review/sprint-93-summary.md:30]: may want to track somewhere that #2768 is still open. but the initial fix was indeed merged in sprint 93.

File: docs/Sprint-Review/sprint-93-summary.md [hunk 6]
```diff
@@ -0,0 +1,80 @@
+# Sprint 93 Summary
+
+02/14/2024 - 02/27/2024
+
+Velocity (Dev): 18
```
Comment (ADPennington) [inline docs/Sprint-Review/sprint-93-summary.md]: i assume this velocity is still correct?

File: docs/Sprint-Review/sprint-93-summary.md (hunk 5 above)
Comment (reitermb) [reply docs/Sprint-Review/sprint-93-summary.md:30]: Do you think the [roadmap item](https://app.mural.co/t/raft2792/m/raft2792/1675268418098/d56ec0d5415d8cda9c6ae9cae61154bcc4e74bba?wid=2-1705963634552) covers that or that we need another tracker?

Comment (ADPennington) [reply docs/Sprint-Review/sprint-93-summary.md:30]: yeah thats fine to track there.

File: docs/Sprint-Review/sprint-93-summary.md [hunk 7]
```diff
@@ -0,0 +1,80 @@
+# Sprint 93 Summary
+
+02/14/2024 - 02/27/2024
+
+Velocity (Dev): 27
```
Comment (reitermb) [inline docs/Sprint-Review/sprint-93-summary.md:5]: @ADPennington I'm trying to track down the pointing disparity between Zenhub's sprint report and adding up what we finished manually, but this should be correct for the later, if oddly inconsistent with how Zenhub is counting

**Review by ADPennington — APPROVED** [review]
🚀

**Pull request merged**